vite_ruby/lib/vite_ruby/config.rb

```diff
@@ -19,17 +19,26 @@ def host_with_port
 
   # Internal: Path to the manifest files generated by Vite and vite-plugin-ruby.
   def known_manifest_paths
-    [
+    manifest_files = [
       # NOTE: Generated by Vite when `manifest: true`, which vite-plugin-ruby enables.
       'manifest.json',
       # NOTE: Path where vite-plugin-ruby outputs the assets manifest file.
       'manifest-assets.json',
-    ].flat_map { |path|
-      [
-        build_output_dir.join(".vite/#{ path }"), # Vite 5 onwards
-        build_output_dir.join(path), # Vite 4 and below
-      ]
-    }
+    ]
+
+    vite5_manifest_paths = manifest_files.map do |path|
+      build_output_dir.join(".vite/#{path}")
```

Comment:
```suggestion
      build_output_dir.join(".vite/#{ path }")
```

Looks like this repo's RuboCop config wants us to do this. ([RuboCop docs](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/SpaceInsideStringInterpolation))

<img width="1229" alt="Screenshot 2024-03-25 at 15 08 59" src="https://github.com/ElMassimo/vite_ruby/assets/1557529/fd0c6fcc-ee17-416e-8064-bef77041fbb2">

https://github.com/ElMassimo/vite_ruby/blob/369facf440f41162efee825a87d9491ff83a03b8/.rubocop.yml#L72-L73

Reply:
👋 @KieranP, any chance we can get this PR fixed up and rebased? Would be nice to have it ready so it can be merged so it's in the next release, whenever that is. 🙏 🙇‍♂️ 

Don't want to keep production application on cherry-picked fix branch long term. 😅

Reply:
@larouxn  Done and done

Reply:
Hi! is it there an ETA for this to be merged and release? Thanks!